### PR TITLE
Gui max

### PR DESCRIPTION
**Pull request by maxronner**

### CODE REVIEW

File: TestTool/TestTool/Views/QuestionPage.xaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                       FontSize="30"/>
+            <TextBlock x:Name="textBlockCounter"
+                       Text="ex. Question 1 of 15"/>
+            <StackPanel Orientation="Horizontal">
+                <TextBlock x:Name="textBlockQuestion"
+                           Text="Question text goes here"
+                           FontSize="20"/>
+                <Image/>
+            </StackPanel>
+            <!--answer should go here, need help-->
```
Comment (WulffMorgan) [inline TestTool/TestTool/Views/QuestionPage.xaml:31]: Make TODO

**Comment by WulffMorgan** [comment]
Merge failed and "are too complex to resolve in the web editor". Please fix merge conflicts locally

**Comment by maxronner** [comment]
Still conflicts, will try to fix it today

**Comment by maxronner** [comment]
§

**Comment by maxronner** [comment]
Need help. I don't know which version of register page to use, will bring it up in meeting tomorrow

**Pull request merged**